src/insns/acperm_32bit.adoc

```diff
@@ -47,12 +47,12 @@ The rules from <<acperm_rules>> must be followed when removing permissions.
 |  2              | <<c_perm>>   | <<r_perm>> or <<w_perm>>
 |  3  (RV32 only) | <<c_perm>>   | <<r_perm>>
 |  4  (RV32 only) | <<x_perm>>   | <<r_perm>>
-|  5  (RV32 only) | <<w_perm>>   | not(<<c_perm>>) or (<<c_perm>> and not(<<lm_perm>>))
+|  5  (RV32 only) | <<w_perm>>   | not(<<c_perm>>) or <<lm_perm>>
 |  6  (RV32 only) | <<x_perm>>   | <<w_perm>> or <<c_perm>>
-|  7  (RV32 only) | <<x_perm>>   | <<c_perm>> == <<lm_perm>> == <<el_perm>>
-|  8              | <<sl_perm>>  | <<w_perm>> and <<c_perm>>
-|  9              | <<el_perm>>  | <<c_perm>> and <<r_perm>>
-| 10              | <<lm_perm>>  | <<c_perm>> and <<r_perm>>
+|  7              | <<el_perm>>  | <<c_perm>> and <<r_perm>>
+|  8              | <<lm_perm>>  | <<c_perm>> and <<r_perm>>
+|  9  (RV32 only) | <<x_perm>>   | <<c_perm>> == <<lm_perm>> == <<el_perm>>
+| 10              | <<sl_perm>>  | <<w_perm>> and <<c_perm>>
```

Comment:
`10              | <<sl_perm>>  | <<w_perm>> and <<c_perm>>`

Should be

`10              | <<sl_perm>>  | <<c_perm>>` ?